core/editor/config-providers/editor-v2-config-provider.php

```diff
@@ -129,26 +106,48 @@ public function get_additional_template_paths() {
 		return Editor_Common_Configs::get_additional_template_paths();
 	}
 
-	private function get_packages_script_assets() {
-		if ( ! $this->packages_script_assets ) {
-			$this->packages_script_assets = Collection::make( [ static::APP_PACKAGE ] )
-				->merge( static::EXTENSION_PACKAGES )
-				->merge( static::UTIL_PACKAGES )
-				->map_with_keys( function ( $package_name ) {
-					$assets_path = ELEMENTOR_ASSETS_PATH;
-					$script_asset_path = "{$assets_path}js/packages/{$package_name}.asset.php";
+	private function map_packages_to_handles( $packages_data, $deps ) {
+		return array_map( function ( $package_name ) use ( $packages_data ) {
+			if ( ! empty( $packages_data[ $package_name ]['handle'] ) ) {
+				return $packages_data[ $package_name ]['handle'];
+			}
 
-					if ( ! file_exists( $script_asset_path ) ) {
-						return [];
-					}
+			if ( 0 === strpos( $package_name, '@wordpress/' ) ) {
+				return str_replace( '@wordpress/', 'wp-', $package_name );
+			}
 
-					/** @var array{ handle: string, deps: string[] } $script_asset */
-					$script_asset = require $script_asset_path;
+			return $package_name;
+		}, $deps );
+	}
 
-					return [ $package_name => $script_asset ];
+	private function get_packages_data() {
+		if ( ! $this->packages_data ) {
+			// Loading the file that is responsible for registering the packages in the filter.
+			require_once ELEMENTOR_ASSETS_PATH . 'js/packages/loader.php';
+
+			$packages_data = apply_filters( 'elementor/packages/config', [] );
+
+			// Packages that there is no access to the package.json file.
+			$type_exceptions = [
+				'@elementor/ui' => static::PACKAGE_TYPE_UTIL,
+			];
+
+			$this->packages_data = Collection::make( $packages_data )
+				->map_with_keys( function ( $data, $name ) use ( $packages_data, $type_exceptions ) {
+					$type = $type_exceptions[ $name ] ?? $data['type'] ?? static::PACKAGE_TYPE_EXTENSION;
+
+					return [
+						$name => [
+							'handle' => $data['handle'],
+							'src' => $data['url'] . $data['entry'] . '{{MIN_SUFFIX}}.js',
```

Comment:
do not add `MIN_SUFFIX` if in debug mode

Reply:
Already handled.